### ISSUE TITLE
ADD --chown UID:GID dan.tar

### ISSUE THREAD

**Issue by rhatdan** [openshift/imagebuilder#77]
Docker build now supports a chown operator on the ADD Function.

```
cat Dockerfile
FROM fedora
COPY --chown 3267:3267 dan.tar
CMD /bin/bash
```

**Comment by smarterclayton** [comment]
I'll have to double check whether docker cp respects the uid in the tar.  If it doesn't, we'll have to mark this under the incompatible syntax flag and error if you specify it.

**Comment by rhatdan** [comment]
@smarterclayton What was the conclusion on this?

**Comment by adambkaplan** [comment]
This blocks dockerless builds in origin.

See projectatomic/buildah#407 and https://jira.coreos.com/browse/DEVEXP-124

**Comment by TomSweeneyRedHat** [comment]
This can be closed as #93 addresses this issue.
